app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -938,36 +939,47 @@ private String getExtraHead() {
         return extra;
     }
 
-    private String getHeader() {
-        StringBuilder header = new StringBuilder();
+    private String getStats() {
+        StringBuilder stats = new StringBuilder();
 
         Date publishedAt = article.getPublishedAt();
         if (publishedAt != null) {
-            header.append("<span style=\"white-space: nowrap;\">");
-
-            header.append(android.text.format.DateFormat.getDateFormat(this).format(publishedAt))
-                    .append(' ')
-                    .append(android.text.format.DateFormat.getTimeFormat(this).format(publishedAt));
-
-            header.append("</span>");
+            stats.append("<li>");
+            stats.append("\t<i class=\"material-icons no-tss\">today</i>");
+            stats.append(android.text.format.DateFormat.getDateFormat(this).format(publishedAt))
+                 .append(' ')
+                 .append(android.text.format.DateFormat.getTimeFormat(this).format(publishedAt));
```

Comment:
This is a very minor issue, but the indentation here is wrong (5 spaces (compared to the previous line) instead of 8).